### ISSUE TITLE
Request: Show a "Polyfill" indicator while presenting

### ISSUE THREAD

**Issue by klausw** [googlevr/webvr-polyfill#300]
Feature request: while presenting using the Polyfill, can you show a small "Polyfill" text somewhere onscreen? 

I've run into several cases where people where expecting to be using native WebVR, but were actually using the polyfill, or where they weren't sure which mode they were in. In a way, the emulation is too transparent.

I think it would be helpful if there's a small onscreen indicator showing that the polyfill is active. This would be especially helpful for troubleshooting, it's a lot easier to ask someone to look for text as opposed to asking them to check for subtle differences in how the settings screen looks.

**Comment by jsantell** [comment]
I like this idea. I've gotten used to detecting the difference between daydream and cardboard UIs, but maybe a more obvious "P" would be helpful to work in a question "do you have [this icon/symbol] on your UI?". Do you think it'd be necessary to convey in whatever token that the polyfill is running? Would that be something most users understand what's going on, or more for debugging?

**Comment by klausw** [comment]
Yes, a "P" for Polyfill would work fine for this purpose. For bonus points, maybe tapping it could pop up an "About" page showing "Polyfill version x.y"?

I think it's dual use, both for debugging ("look for the 'P'") , and as something users can recognize more easily than looking for subtle styling differences.
